Libplanet.Explorer.Executable/Program.cs

```diff
@@ -30,11 +29,135 @@ namespace Libplanet.Explorer.Executable
     /// <summary>
     /// The program entry point to run a web server.
     /// </summary>
-    public class Program
+    public class Program : CoconaLiteConsoleAppBase
     {
         public static async Task Main(string[] args)
         {
-            Options options = Options.Parse(args, Console.Error);
+            await CoconaLiteApp.RunAsync<Program>(args);
+        }
+
+        [Command(Description = "Run libplanet-explorer with options.")]
+        public async Task Run(
+            [Option(
+                "seed",
+                new[] { 's' },
+                Description = @"Seed nodes to join to the network as a node. The format of each
+seed is a comma-separated triple of a peer's hexadecimal public key, host, and port number.
+E.g., `02ed49dbe0f2c34d9dff8335d6dd9097f7a3ef17dfb5f048382eebc7f451a50aa1,example.com,31234'.
+If omitted (default) explorer only the local blockchain store.")]
+            string[] seedStrings,
```

Comment:
Is it okay to be _required_ option? In my memory, libplanet-explorer could be run in local (i.e. offline mode), not in network, to explore local storage.

Reply:
Ah I understand. I'll fix this accordingly.